cms/templates/home_page.html

```diff
@@ -19,16 +19,22 @@
 {% endblock %}
 
 {% block content %}
-  {% include "partials/hero.html" with action_title="Watch Now" action_url="#" %}
-  {% if page.upcoming_courseware %}
-    {% pageurl catalog_page as catalog_url %}
-    {% include "partials/course-carousel.html" with page=page.upcoming_courseware button_title="View All" button_url=catalog_url %}
-  {% endif %}
-  {% if page.about_mit_xpro %} {% include "partials/text-video-section.html" with page=page.about_mit_xpro %} {% endif %}
-  {% if page.learning_experience %} {% include "partials/learning-techniques.html" with page=page.learning_experience %} {% endif %}
-  {% if page.testimonials %} {% include "partials/testimonial-carousel.html" with page=page.testimonials %} {% endif %}
-  {% if page.inquiry_section %} {% include "partials/for-teams.html" with page=page.inquiry_section %} {% endif %}
-  {% if page.image_carousel_section %} {% include "partials/image-carousel.html" with page=page.image_carousel_section %} {% endif %}
+  {% with background_video_url=page.background_video_url upcoming_courseware=page.upcoming_courseware %}
```

Comment:
are we utilizing the var background_video_url in home page ??

Reply:
Yes.